spec/project_spec.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 
-describe 'RuboCop Project' do # rubocop:disable RSpec/DescribeClass
+describe 'RuboCop Project' do
```

Comment:
Yeah interestingly I actually need that to make it pass locally, due to
different versions of Rubocop I guess.

On Wednesday, September 16, 2015, Andy Waite <notifications@github.com>
wrote:

> In spec/project_spec.rb
> <https://github.com/nevir/rubocop-rspec/pull/49#discussion_r39669366>:
>
> > @@ -1,6 +1,6 @@
> >  # encoding: utf-8
> >
> > -describe 'RuboCop Project' do # rubocop:disable RSpec/DescribeClass
> > +describe 'RuboCop Project' do
>
> This change seems be causing Travis to fail.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nevir/rubocop-rspec/pull/49/files#r39669366>.
>